src/loc.rs

```diff
@@ -56,10 +56,27 @@ impl Loc {
     }
 }
 
+/// The set of errors which may occur.
+///
+/// This enum describes the possible errors when parsing a loc.  A valid loc
```

Comment:
Yes and I recalled it such that I reverted most of them later on.

It is still intuitive for me to do the double spacing so I am going to revert all of them with a new commit.  ONE commit for all reversions in all source files and the README.

Reply:
The double spacing just occurs once in line 27 of the README.  I am going to combine this change with another one since I assume it would be considered too small for a commit on its own.

If I should have overseen other occurrences in `main.rs` or `lib.rs`, please tell me.